tests/pytests/test_tdscf_excitations.py

```diff
@@ -24,1262 +23,218 @@
+    pytest.param(  h2o, 'RHF-1',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, RPA]),
+    pytest.param(  h2o, 'RHF-1',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, TDA]),
+    pytest.param(  h2o, 'RHF-3',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, RPA]),
+    pytest.param(  h2o, 'RHF-3',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, TDA]),
+    pytest.param( h2o2, 'RHF-1',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, RPA]),
+    pytest.param( h2o2, 'RHF-1',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, TDA]),
+    pytest.param( h2o2, 'RHF-3',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, RPA]),
+    pytest.param( h2o2, 'RHF-3',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, TDA]),
+    pytest.param( moxy, 'RHF-1',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, RPA]),
+    pytest.param( moxy, 'RHF-1',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_singlet, TDA]),
+    pytest.param( moxy, 'RHF-3',     'wB97X',  'RPA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, RPA]),
+    pytest.param( moxy, 'RHF-3',     'wB97X',  'TDA',  'cc-pvdz', marks=[hyb_gga_lrc, RHF_triplet, TDA]),
+], ids=_test_id) # yapf: disable
+def test_tdscf(mol, ref, func, ptype, basis):
+    # expected failures
+    if (ref == 'RHF-3') and (func != "HF"):
         pytest.xfail("RKS Vx kernel only Spin Adapted for Singlet")
     elif (ref == 'UHF' and func != 'SVWN'):
-        pytest.xfail("UKS Vx kernel bug for non-lda")
-
-    ### setup
-    # Look up expected in fixture (easier to read)
-    exp_lookup = "{}-{}-{}-{}".format(ref, func, ptype, basis)
-    exp_low_per_sym = {'A1': 100, 'A2': 100, 'B1': 100, "B2": 100}
-    for x in expected[exp_lookup]:
-        exp_low_per_sym[x['sym']] = min(x['e'], exp_low_per_sym[x['sym']])
-
-    exp_energy_sorted = np.array([x['e'] for x in expected[exp_lookup]])
-    exp_energy_sorted.sort()
+        pytest.xfail("UKS Vx kernel bug for non-LDA")
 
-    # get wfn (don't use symmetry b/c too slow)
-    wfn = wfn_factory(ref, func, basis, nosym=True)
-    # select solver function (TDA->davidson/RPA->hamiltonian)
-    solver = solver_funcs[ptype]
-
-    # build engine
-    engine = engines[ref](wfn, ptype)
-
-    # skipping the entrypoint, just call the solver
-    out = solver(
-        engine=engine,
-        guess=engine.generate_guess(16),
-        max_vecs_per_root=10,
-        nroot=4,
-        verbose=1,
-        maxiter=30,
-        e_tol=1.0e-7,
-        r_tol=1.0e-5,
-        schmidt_tol=1.0e-12)
-
-    test_vals = out[0]
-    stats = out[-1]
-    assert stats[-1]['done'], "Solver did not converge"
-
-    for i, my_v in enumerate(test_vals):
-        ref_v = exp_energy_sorted[i]
-        assert compare_values(ref_v, my_v, 4, "{}-{}-{}-ROOT-{}".format(ref, func, ptype, i + 1))
+    psi4.core.clean()
```

Comment:
this is called between each test anyways. is it misbehaving that needed again here?

Reply:
copy-paste from previous iteration on these tests.